example/integration_test/plugin_integration_test.dart

```diff
@@ -0,0 +1,22 @@
+// This is a basic Flutter integration test.
+//
+// Since integration tests run in a full Flutter application, they can interact
+// with the host side of a plugin implementation, unlike Dart unit tests.
+//
+// For more information about Flutter integration tests, please see
+// https://docs.flutter.dev/cookbook/testing/integration/introduction
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'package:flutter_eco_mode/flutter_eco_mode.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('getPlatformVersion test', (WidgetTester tester) async {
+    final FlutterEcoMode plugin = FlutterEcoMode();
```

Comment:
What's the purpose of the integration test ? It seems to be empty ? Maybe we could delete this part ?